src/main.cpp

```diff
@@ -5157,22 +5157,36 @@ void static ProcessGetData(CNode* pfrom, const Consensus::Params& consensusParam
                     }
                 }
 
-                if (!pushed && inv.type == MSG_GOVERNANCE_OBJECT) {
-                    if(governance.mapObjects.count(inv.hash)) {
-                        CDataStream ss(SER_NETWORK, PROTOCOL_VERSION);
-                        ss.reserve(1000);
-                        ss << governance.mapObjects[inv.hash];
-                        pfrom->PushMessage(NetMsgType::MNGOVERNANCEOBJECT, ss);
+                if (!pushed && inv.type == MSG_GOVERNANCE_OBJECT_VOTE) {
```

Comment:
please keep `if (!pushed && inv.type == MSG_GOVERNANCE_OBJECT_VOTE) {`-like blocks in the same order net messages are - swap them back i.e. `MSG_GOVERNANCE_OBJECT` first, then `MSG_GOVERNANCE_OBJECT_VOTE`